rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -406,7 +409,9 @@ private String getGeneratedNodePrivateKey() {
                 file.getParentFile().mkdirs();
                 try (FileWriter writer = new FileWriter(file)) {
                     props.store(writer, "Generated NodeID. To use your own nodeId please refer to 'peer.privateKey' config option.");
-                    logger.info("New nodeID generated: {}", props.getProperty("nodeId"));
+                    if(logger.isInfoEnabled()) {
+                        logger.info("New nodeID generated: {}", props.getProperty("nodeId"));
+                    }
                     logger.info("Generated nodeID and its private key stored in {}", file);
```

Comment:
Isn't this info supposed to be added to the ìf` as well, since you refactored to add the verification of this before log?

Reply:
Yep, I did it automatically to fix a sonar issue and didn't realize about it.